dotnet-template-samples/README.md

```diff
@@ -0,0 +1,30 @@
+# .NET Template Samples
```

Comment:
this readme has to be adjusted. We can skip it for now, on adjust with suggestions below.

Reply:
done